contracts/cw721-non-transferable/src/query.rs

```diff
@@ -0,0 +1,10 @@
+use crate::{msg::GetAdminResponse, state::CONFIG};
+use cosmwasm_std::{Deps, Env, StdResult};
+
+pub fn admin(deps: Deps, _env: Env) -> StdResult<GetAdminResponse> {
```

Comment:
`_env` can be removed if it’s not used in the function

Reply:
done, will take it into account from now on